src/components/Modal/NewDashboardModal/index.tsx

```diff
@@ -50,9 +54,11 @@ function NewDashboardModal({ showModal, handleClose }: NewDashboardModalProps) {
 
   const handleNewDashboard = async (data: FormValues) => {
     try {
-      await httpClient.post('/dashboards', data);
+      const res = await axios.post('/dashboards', data);
       setToast('success', '대시보드가 생성되었습니다.');
       handleResetClose();
+      setSelectDashboard(res.data);
```

Comment:
```suggestion
      setSelectDashboard(res.data);
```
대시보드/id 페이지에서 ssr로 대시보드 정보 가져오고 로컬 스토리지에 동기화해서 이 코드 없어도 문제 없을 것 같네요!